generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -2,61 +2,84 @@
 image: openjdk:8
 <%_ } _%>
 
-<%_ if (buildTool == 'gradle') { _%>
+<%_ if (buildTool === 'gradle') { _%>
 cache:
     key: "$CI_BUILD_REF_NAME"
     paths:
         - node_modules
         - .gradle/wrapper
         - .gradle/caches
 <%_ } _%>
-<%_ if (buildTool == 'maven') { _%>
+<%_ if (buildTool === 'maven') { _%>
 cache:
     key: "$CI_BUILD_REF_NAME"
     paths:
         - node_modules
-        - .maven
+        - .m2
```

Comment:
We do an `export MAVEN_USER_HOME=`pwd`/.maven` so all 3rd party artifacts will go into `maven` and not `.m2` I think, so caching won't work.